### PR TITLE
Build validator

### LINKED ISSUE #1: Create Validator Component
- Reaches out to the server and download a few ``QuestionAnswerPair``.
- Allows user to edit the question, answer, and type .
- Has delete button which erases question from server
- Has submit button which updates ``QuestionAnswerPair`` in server.

### PR DESCRIPTION
**Pull request by thegooman**
Features:
- [x] Able to get queries from the db
- [x] Update queries upon submit
- [x] Delete invalid queries

### CODE REVIEW

**Comment by thegooman** [comment]
@Waidhoferj if you could take a look at this and provide feedback, mason and I would greatly appreciate it!

File: src/pages/Validator/Validator.js [hunk 1]
```diff
@@ -1,94 +1,88 @@
...
   let [selectedIndex, setSelectedIndex] = useState(0);
 
-  let deleteCurrentQuery = () => {
+  let deleteCurrentQuery = async () => {
     let updatedQueries = [...queries];
     updatedQueries.splice(selectedIndex, 1);
 
+    // delete query in database
+    axios.post(`${baseUrl}/new_data/delete_phrase`);
+
```
Comment (Waidhoferj) [inline src/pages/Validator/Validator.js]: There's an easy way to configure axios with a base url [globally](https://css-tricks.com/stay-dry-using-axios-for-api-requests/)

File: src/pages/Validator/Validator.js [hunk 2]
```diff
@@ -1,94 +1,88 @@
...
     });
     updatedQueries.push(...response);
     setQueries(updatedQueries);
     //Adjust the selected index
     setSelectedIndex((i) => i - dequeueCount);
   };
+  useEffect(() => {
+    fetchMoreQueries(3, queries);
+  }, []);
+  // TODO: Build a loading page while queries are initially fetched
```
Comment (Waidhoferj) [inline src/pages/Validator/Validator.js:83]: Great use case for a Suspense component!

**Review by Waidhoferj — APPROVED** [review]
Awesome job bringing it full circle @kgoo124 ! The code looks great!

File: src/pages/Validator/Validator.js (hunk 2 above)
Comment (Waidhoferj) [reply src/pages/Validator/Validator.js:83]: *Referencing the comment for a later PR

File: src/pages/Validator/Validator.js (hunk 1 above)
Comment (masonmcelvain) [reply src/pages/Validator/Validator.js]: Good idea! Done :)

**Pull request merged**